safenode/src/network/mod.rs

```diff
@@ -133,9 +126,18 @@ impl SwarmDriver {
             .map(|(peer_id, muxer), _| (peer_id, StreamMuxerBox::new(muxer)))
             .boxed();
 
-        // Create a Kademlia behaviour for client mode, i.e. set req/resp protocol
-        // to outbound-only mode and don't listen on any address
+        // Kademlia configuration
+        let mut cfg = KademliaConfig::default();
+        // The `replication_factor` determines the a) The number of closer peers
+        // returned in response to a request b) The number of closest peers to a
+        // key to search for in an iterative query.
+        // It does not affect the maximum number of nodes in a bucket; this is set
+        // by `kad::K_VALUE`.
+        let replication_factor =
+            NonZeroUsize::new(CLOSE_GROUP_SIZE).ok_or(Error::NonZeroNumberExpected)?;
```

Comment:
I wonder if we could make the close group a `static` `NonZeroUsize` to avoid this `ok_or` check (which should always be unnecessary).

(it can't be made a `const` due to fn call)

Shouldn't botch the use of `CLOSE_GROUP_SIZE` in other places, I think?